pages/common/n.md

```diff
@@ -0,0 +1,24 @@
+# n
+
+> Node version management.
```

Comment:
Maybe better to say - "Tool to manage multiple node versions". Node version management can mean a lot of things.